Python/ceval.c

```diff
@@ -1962,44 +1962,6 @@ initialize_locals(PyThreadState *tstate, PyFunctionObject *func,
     return -1;
 }
 
-/* Consumes references to func, locals and all the args */
```

Comment:
Why was this moved downwards?

Reply:
There was a small change, it now calls `clear_thread_frame()` rather than `_PyEvalFrameClearAndPop()`, so now needs to come after `clear_thread_frame`